src/suggestions.js

```diff
@@ -212,8 +204,26 @@ Suggestions.prototype.getCandidates = function(callback) {
  * @param {Object|String} item an item from the data array
  * @return {String} item
  */
-Suggestions.prototype.getItemValue = function(item) {
+Suggestions.prototype.getItemValue = function (item) {
   return item;
 };
 
-module.exports = Suggestions;
+/**
+ * For a given item in the data array, return a string of html that should be rendered in the dropdown
+ * @param {Object|String} item an item from the data array
+ * @return {String} html
+ */
+Suggestions.prototype.render = function (item) {
+  var boldString = this.getItemValue(item);
+  var indexString = this.normalize(boldString);
+
+  var indexOfQuery = indexString.lastIndexOf(this.query);
+  while (indexOfQuery > -1) {
+    var endIndexOfQuery = indexOfQuery + this.query.length;
+    boldString = boldString.slice(0, indexOfQuery) + '<strong>' + boldString.slice(indexOfQuery, endIndexOfQuery) + '</strong>' + boldString.slice(endIndexOfQuery);
```

Comment:
Not sure I'm following this. if this `render` method allows a user to template a list item, would wrapping `<strong>` be desired or expected?

Reply:
If the user provides a custom `render` method, then that will be used instead. If the user does not provide a custom render function, then this default function will be used.  The default function follows the existing functionality of wrapping the matching text in <strong> tags.

Reply:
oh dear me 🙈 I see now the existing functionality in the diff. 👍